bin/set-wire-server-image-version.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+USAGE="$0 <target-backend-version>"
+target_version=${1?$USAGE}
+
+charts=(brig cannon galley gundeck spar cargohold proxy cassandra-migrations)
+
+for chart in "${charts[@]}"; do
+    sed -i "s/  tag: .*/  tag: $target_version/g" "charts/$chart/values.yaml"
+done
+
+#special case nginz
+sed -i "s/   tag: .*/   tag: $target_version/g" "charts/nginz/values.yaml"
```

Comment:
What's different here, number of spaces, right? Would it make sense to use a tool like `yaml`? Or are we trying to avoid more dependencies?

Reply:
Just number of spaces.

We're trying to avoid more dependencies - this script was originally written on a system without `yaml` available (but `sed` was available).